app/src/main/java/com/example/vande/scouting2017/TeleopActivity.java

```diff
@@ -1,13 +1,321 @@
+//                headingDataStringList.add("autonLowFuel");
+//                headingDataStringList.add("activatedHopper");
+//                headingDataStringList.add("teleopGearPlaced");
+//                headingDataStringList.add("gearPlacement");
+//                headingDataStringList.add("teleopGearDropped");
+//                headingDataStringList.add("teleopHighFuelScored");
+//                headingDataStringList.add("teleopHighFuelMissed");
+//                headingDataStringList.add("teleopLowFuel");
+//                headingDataStringList.add("fuelRetrieval");
+//                headingDataStringList.add("gearRetrieval");
+//                headingDataStringList.add("climbing");
+//                headingDataStringList.add("climbTime");
+//                headingDataStringList.add("defense");
+//            }
+//            else{
+//                //empty header as the are already created
+//                headingDataStringList.add("");
+//            }
+
+            teleopDataStringList.add(getTextInputLayoutString(teleopGearPlacedInputLayout));
+            teleopDataStringList.add(gearPlacement_Radiobtn.getText());
+            teleopDataStringList.add(getTextInputLayoutString(teleopGearDroppedInputLayout));
+            teleopDataStringList.add(getTextInputLayoutString(teleopHighFuelScoredInputLayout));
+            teleopDataStringList.add(getTextInputLayoutString(teleopHighFuelMissedInputLayout));
+            teleopDataStringList.add(getTextInputLayoutString(teleopLowFuelInputLayout));
+            teleopDataStringList.add(fuelRetreival_Radiobtn.getText());
+            teleopDataStringList.add(gearRetreival_Radiobtn.getText());
+            teleopDataStringList.add(climbing_Radiobtn.getText());
+            teleopDataStringList.add(getTextInputLayoutString(climbTimeInputLayout));
+            teleopDataStringList.add(defense_Radiobtn.getText());
+
+            String message = auton +","+ FormatStringUtils.addDelimiter(teleopDataStringList, ",");
+//            String message = FormatStringUtils.addDelimiter(teleopDataStringList, ",") + auton +","+ FormatStringUtils.addDelimiter(teleopDataStringList, ",");
+
+//                message = heading + auton +","+ teleop + "\n";
+                //Output data to file
+                try{
+                    FileOutputStream fileOutputStream= new FileOutputStream(file,true);
+                    fileOutputStream.write(message.getBytes());
+                    fileOutputStream.close();
+                } catch (IOException e){
+                    e.printStackTrace();
+                }
+            }else{
+                Toast.makeText(getApplicationContext(),"SD card not found", Toast.LENGTH_LONG).show();
+            }
+            Toast.makeText(getApplicationContext(),"message Saved", Toast.LENGTH_LONG).show();
+
+
+                Intent intent = new Intent(this, AutonActivity.class);
+                startActivity(intent);
```

Comment:
This is going to cause multiple activities

Reply:
removed start Activity and implemented on activity result